custom_components/ocpp/button.py

```diff
@@ -0,0 +1,92 @@
+"""Button platform for ocpp."""
+from dataclasses import dataclass
+from typing import Final
+
+from homeassistant.components.button import (
+    DOMAIN as BUTTON_DOMAIN,
+    ButtonDeviceClass,
+    ButtonEntity,
+    ButtonEntityDescription,
+)
+from homeassistant.helpers.entity import DeviceInfo, EntityCategory
+
+from .api import CentralSystem
+from .const import CONF_CPID, DEFAULT_CPID, DOMAIN
+from .enums import HAChargerServices
+
+
+@dataclass
+class OcppButtonDescription(ButtonEntityDescription):
+    """Class to describe a Button entity."""
+
+    press_action: str | None = None
```

Comment:
I don't understand this | None syntax.

Reply:
Neither did I, I copied it over from the Wallbox integration but looks like it does not work with a string. Should be fixed now 👍

Reply:
Looks like the issue was I forgot to import typing Any. See Union [here](https://docs.python.org/3/library/stdtypes.html#types-union) for explanation